Submodels/MTFLO_call.py

```
@@ -1,20 +1,183 @@
+        Requires that the executable, mtflo.exe, and the input file, tflow.xxx are present in the same directory as this
+        Python file. 
+        """
+
+        # Get the directory where the current Python file is located
+        current_file_directory = os.path.dirname(os.path.abspath(__file__))
+
+        # Change the working directory to the directory of the current Python file
+        os.chdir(current_file_directory)
+
+        # Generate the subprocess and write it to self
+        self.process = subprocess.Popen([self.fpath, self.analysis_name], 
```

Ruff (0.8.2)

`subprocess` call with `shell=True` identified, security issue

(S602)